src/script/components/Conversation/Conversation.tsx

```diff
@@ -460,6 +464,19 @@ export const Conversation = ({
     [addReadReceiptToBatch, repositories.conversation, repositories.integration, updateConversationLastRead],
   );
 
+  const onGoToLastMessage = () => {
+    if (activeConversation) {
+      activeConversation.initialMessage(undefined);
```

Comment:
clean up anything like search result

Reply:
This would be a valuable comment to add to the code directly

Reply:
done